CRM/Exthours/Upgrader.php

```diff
@@ -68,6 +68,12 @@ public function uninstall() {
       ->addWhere('label', '=', 'Tracking Number')
       ->execute();
 
+    // Delete all related custom field
+    $cleanCustomField = \Civi\Api4\CustomField::delete()
+      ->setCheckPermissions(FALSE)
+      ->addWhere('label', '=', 'Is Invoiced?')
```

Comment:
@lucky091588 Because labels can change, please use `name` here instead of `label`.